.NET/Microsoft.Recognizers.Text.DateTime/English/Extractors/EnglishDateExtractorConfiguration.cs

```diff
@@ -196,12 +199,9 @@ public EnglishDateExtractorConfiguration(IOptionsConfiguration config)
 
             DateRegexList = new List<Regex>
             {
-                // ((this)? Sunday,)? April 5
+                // (Sunday,)? April 5
```

Comment:
Please change all comments for the merged regex to // (Sunday,)? April 5 or (Sunday,)? April 5, 2016 
to illustrate both are covered together.